spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestViews.java

```diff
@@ -2114,6 +2114,22 @@ public void createViewWithCustomMetadataLocation() {
                 ""));
   }
 
+  @TestTemplate
```

Comment:
Is the reason for not backporting to v3.4 that's deprecated #14099?

Reply:
Yes, I don't think it's necessary to backport to v3.4.